media/css/firefox/privacy/products.scss

```diff
@@ -225,7 +225,7 @@ $image-path: '/media/protocol/img';
 }
 
 .privacy-products-pocket .mzp-c-card-feature-title {
-    background-image: url('/media/protocol/img/logos/pocket/logo-word-horz.svg');
+    background-image: url('/media/protocol/img/logos/pocket/logo-word-hor.svg');
```

Comment:
Needs an explicit background size declaration to go with it (see screen shot in other comment).